karma.conf.js

```diff
@@ -77,5 +84,11 @@ module.exports = function(config) {
     // Concurrency level
     // how many browser should be started simultaneous
     concurrency: Infinity
-  });
+  };
+
+  if (process.env.TRAVIS) {
```

Comment:
No. This will only be available on Travis and would return `undefined` locally which is what we want.